module/config/argparser.py

```diff
@@ -77,6 +77,11 @@ def main(ini_name=''):
     event_latest = sorted([f for f in event_folder], reverse=True)[0]
     event_folder = [dic_ini_to_gui.get(f, f) for f in event_folder][::-1]
     event_latest = dic_ini_to_gui.get(event_latest, event_latest)
+    # Archives list
+    archives_folder = [f for f in dic_event[server.server].values() if f.startswith('war_archives_')]
+    archives_latest = sorted([f for f in archives_folder], reverse=True)[0]
+    archives_folder = [dic_ini_to_gui.get(f, f) for f in archives_folder][::-1]
+    archives_latest = dic_ini_to_gui.get(archives_latest, archives_latest)
```

Comment:
Since I haven't unlocked all the stages yet I did do practice runs on earlier stages that did have sirens. The siren template 'DD' and 'CL' for Winter's Crown and Fallen Wings worked. Divergent Chessboard doesn't have any sirens according to the wiki.

Do you still prefer I delete these EX maps?

Reply:
Sorry, word `EX` is ambiguous. I mean maps like `campaign/war_archives_20181026_en/ex.py` and `campaign/war_archives_20191031_en/ex.py`